src/components/Table.tsx

```diff
@@ -50,6 +53,7 @@ const BaseTable = styled.table`
 	> tbody {
 		> tr {
 			> td {
+				text-align: left;
```

Comment:
**In a future releast**, this will probably have to be configurable as part of the column and as a fallback be inferred by the data type (if it's a primitive). eg: numeric columns would be aligned right by default.

I could reason that using a custom column render function gives us huge power, but it might be an overkill if you have to use it just for aligning stuff. Sensible defaults are also attractive :)

PS: this is probably not the place/PR to start such discussion.

Reply:
@thgreasi Interesting, this is definitely something to think about as we move forward with he project